sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -151,18 +154,24 @@ export class AssessmentControllerService {
    * Get estimated and actual(if present) award details of an assessment.
    * @param assessmentId assessment to which awards details belong to.
    * @param includeDocumentNumber when true document number is mapped
-   * to disbursement dynamic data.
-   * @param studentId studentId student to whom the award details belong to.
+   * to disbursement dynamic data.\
+   * @param options options,
+   * - `studentId` studentId student to whom the award details belong to.
+   * - `applicationId` applications id.
```

Comment:
Maybe we can add more information to the comments, for instance, that the `applicationId` is used for authorization purposes in this case.
```ts
* - `applicationId` application is used for authorization purposes to ensure that a user has access to the specific application data.
```